test/distributed/test_c10d.py

```diff
@@ -578,6 +578,18 @@ def test_nominal(self):
         # check with get
         self.assertEqual(b"value0", store0.get("key0"))
 
+    @retry_on_connect_failures
+    def test_logging_init(self):
+        os.environ["WORLD_SIZE"] = "1"
+        os.environ["MASTER_ADDR"] = "127.0.0.1"
+        os.environ["MASTER_PORT"] = str(common.find_free_port())
+        os.environ["RANK"] = "0"
+
+        self.assertFalse(logging.root.hasHandlers())
```

Comment:
This is not necessarily False in every environment. I think we should assert that the handlers before and after init_process_group are the same.

Reply:
@pritamdamania87 I modified the test to check individual handlers before and after the call to init_process_group